packages/oceanfront/src/fields/Color.ts

```diff
@@ -40,10 +50,26 @@ export const ColorField = defineFieldType({
     const initialValue = computed(() => {
       let initial = ctx.initialValue
       if (initial === undefined) initial = props.defaultValue
+
       return initial ?? null
     })
+
+    const types = ['hex', 'hsl', 'rgb']
+
+    const colorMode = computed(() => {
+      let result = props.record?.metadata?.colorMode
+        ? props.record?.metadata?.colorMode
```

Comment:
We know the properties are set here, so just `props.record.metadata.colorMode` could be used. However it would be more efficient to write `let result = props.record?.metadata?.colorMode || props.context`